A3ServerTool/Models/Config/BasicConfig.cs

```diff
@@ -104,9 +104,33 @@ public class BasicConfig : IConfig
         /// <summary>
         /// Maximum packet size
         /// </summary>
-        [ConfigProperty(PropertyName = "maxPacketSize")]
+        [ConfigProperty(PropertyName = "maxPacketSize", Parent = "sockets")]
         [WrappingClass(new string[] { "sockets" })]
-        public int? MaxPacketSize { get; set; } = 1400;
+        public int? SocketsMaxPacketSize { get; set; } = 1400;
```

Comment:
Its been discussed on the Arma 3 Discord that increasing this value to `1440` fixes some network message issues

Reply:
Yeah, let's increase. I saw dedmen's comment, and he is an absolute expert in these cases. 😃